src/test/java/seedu/address/logic/commands/ExportCommandTest.java

```diff
@@ -74,13 +74,6 @@ public void execute_unableToCreateCsvDirectory_throwsCommandException() throws E
         assertEquals("Could not create directory for CSV file.", thrown.getMessage());
     }
 
-    @Test
```

Comment:
Great catch figuring out that the test case would not affect code coverage due to the way that the user prefs are created as well as the functions to change the addressbook address!